nncf/experimental/openvino_native/graph/node_utils.py

```diff
@@ -155,7 +155,7 @@ def get_reduce_op(node: ov.Node, output_port_id: int) -> ov.Node:
         name_output_port_id = output_port_id
         if reduction_axes_ is None:
             partial_shape = get_partial_shape_safe(node, output_port_id)
-            reduction_axes_ = np.array(range(partial_shape.rank.get_length()))
+            reduction_axes_ = np.arange(partial_shape.rank.get_length()).astype(np.int64)
```

Comment:
Are you sure that every `op` supports `np.int64` type?

Reply:
Yes, it's covered by tests and described [here](https://docs.openvino.ai/latest/openvino_docs_ops_reduction_ReduceMean_1.html) 
```
T_IND : any supported integer type.
```